### PR TITLE
Updated the repository :- Added my details to fulfill the task :)

### PR DESCRIPTION
**Pull request by ShahaamMarzook**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [x] Included a Preview link and screenshot showning after and before the changes.
- [x] Images are `240 x 240` [w x h].
- [x] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

**Comment by ShahaamMarzook** [comment]
Pleases accept my pr to fulfil my task ;
https://codein.withgoogle.com/dashboard/task-instances/4647505826414592/

ASAP
With respect,
Shahaam Marzook
before and after updating my details in students yml
before and after updating the picture in img file

File: _data/students.yml [hunk 1]
```diff
@@ -303,6 +303,12 @@
   github: Akharote1
   image: aditya_kharote.jpg
   twitter: Akharote1
+  
+- name: Shahaam Marzook
+  country: SriLanka
+  github: ShahaamMarzook
+  image: Shahaam.jpg
+  twitter: @mohammedshahaam
```
Comment (nikit19) [inline _data/students.yml]: Please remove the @ from your twitter username

**Review by nikit19 — CHANGES_REQUESTED** [review]
Please resolve conflicts

File: _data/students.yml (hunk 1 above)
Comment (ShahaamMarzook) [reply _data/students.yml]: @nikit19 
Thanks for pointing the mistake.Thank you
I have corrected it has you pointed it.
 
With respect,
Shahaam Marzook

**Review by nikit19 — APPROVED** [review]

**Review by anshumanv — APPROVED** [review]
LGTM but it would've been great if you could squash your commits.

**Pull request merged**